src/core/js/accessibility.js

```diff
@@ -129,7 +129,25 @@ define([
             });
 
             Handlebars.registerHelper('a11y_attrs_heading', function(level) {
```

Comment:
```js
Handlebars.registerHelper('a11y_attrs_heading', function(level, defaultName) {
```
something like ^ that might be a better strategy. it'd be more clear where the values are coming from

```js
                var cfg = Adapt.config.get('_accessibility');

                // if a level has not been passed as a parameter
                if (!_.isString(level)) {
                    level = 0;
                    // look in the global configuration
                    if (cfg._ariaLevels && cfg._ariaLevels['_'+defaultName] !== undefined) {
                        level = cfg._ariaLevels['_'+defaultName]
                    }
                }
                return level ? ' role="heading" aria-level="'+level+'" '+getTabIndex()+' ' : getTabIndex()+' ';
```